core/templates/dev/head/pages/base.html

```diff
@@ -265,7 +265,7 @@
                               </div>
                             </li>
 
-                            <li class="dropdown oppia-navbar-clickable-dropdown pull-right">
+                            <li class="dropdown oppia-navbar-clickable-dropdown pull-right about-oppia-list-item">
```

Comment:
For protractor tests, use a class starting with protractor-test, e.g. protractor-test-about-oppia-list-item.